scene/resources/packed_scene.cpp

```diff
@@ -344,6 +345,12 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 					} else {
 						Variant value = props[nprops[j].value];
 
+						// Making sure that instances of inherited scenes don't share the same
+						// value between them.
```

Comment:
I would say "reference" here instead of "value", since dictionaries and array types in godot are not value types.